### ISSUE TITLE
Issue with update to 1045

### ISSUE THREAD

**Issue by jackmeat** [Jackett/Jackett#9412]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**: windows 10

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**: current

**Jackett Version**:

**Last Working Jackett Version**: 1037

**Are you using a proxy or VPN?** [yes/no] Yes

### Description

Jackett has been informing me it has been updated from  0.16.1037.0  to 0.16.1045 successfully in the ntification, but when I check it, it is still .1037. I attempted to manually trigger the update since I have been getting lots of access denied errors for certain indexers, and figured having the latest version might help. Problem is when manually triggered, it still says completed but I have this in the log:
 
### Logged Error Messages
  {0} System.UnauthorizedAccessException: Access to the path 'clrjit.dll' is denied.    at System.IO.FileSystem.RemoveDirectoryRecursive(String fullPath, WIN32_FIND_DATA& findData, Boolean topLevel)    at System.IO.FileSystem.RemoveDirectory(String fullPath, Boolean recursive)    at Jackett.Common.Services.UpdateService.CleanupTempDir() in D:\a\1\s\src\Jackett.Common\Services\UpdateService.cs:line 218 

### Screenshots

[Place any screenshots of the issue here if needed]
![image](https://user-images.githubusercontent.com/15189742/91652706-189f8180-eadd-11ea-8d05-7838952fe6c9.png)

**Comment by garfield69** [comment]
looks like the Jackett update service is being denied access to the temp directory where the installation files from the previous update archive were extracted to, this in turn is preventing the updater from moving forward.

what could have a lock on that archive? Perhaps anti-malware security?
Check if that security app needs its virus keys updated or even the whole app upgraded. It may be silently flagging a false positive and locking out the updated install package.

In addition, I suggest you try shutting down all Jackett related tasks (the service, tray, anything else in the task manager starting with jackett*.)
Then manually download the latest Jackett windows exe from https://github.com/Jackett/Jackett/releases/download/v0.16.1045/Jackett.Installer.Windows.exe  and install it as a service and see if that completes the install and gets you the expected version.

Then keep an eye for when the next update is released and see if that succeeds.

**Comment by jackmeat** [comment]
Thank you for getting back to me. Don’t know what may have been locking that archive. No additional AV/malware programs running on this PC, just defender. I did as you suggested via the manual install which again returned the same version? But I did go back through and remove jackett as best I could and ran the install again as admin (doubt that had anything to do with it) and it installed just fine. This includes those indexers that were failing as well so I can say case closed. I will keep an eye out for the next time it tries to update and see if it happens again.

 

Not all familiar, do I need to go in and close an issue or anything?

 

Btw, this app is a great piece. Thanks for it, and for the assistance. Appreciate it.
